examples/harmonic-oscillators/harmonic-oscillators.py

```diff
@@ -615,18 +618,18 @@ def generate_pmf_data(ndim=1, nbinsperdim=15, nsamples = 1000, K0=20.0, Ku = 100
   for i in range(numbrellas):
     center = []
     for d in range(ndim):
-      val = gridscale*((i/dp[d]) % nperdim[d] + xrange[d][0])
+      val = gridscale*((int(i//dp[d])) % nperdim[d] + xrange[d][0])
```

Comment:
just to make sure, you intend for this to be floor integer division, correct?

Reply:
Yes!